### PR TITLE
Updated Dutch localization

### PR DESCRIPTION
**Pull request by iMicknl**

### CODE REVIEW

**Comment by corinagum** [comment]
Thanks for the contribution! Please add your username to `LOCALIZATION.md` and resolve merge conflicts :)

**Comment by iMicknl** [comment]
@corinagum I will fix the merge conflicts. My username is already in the `LOCALIZATION.md` from the V3 version.

@compulim should these files Localisation files be excluded from coveralls?

**Comment by compulim** [comment]
@iMicknl I won't look too much into coveralls right now.

**Comment by compulim** [comment]
@corinagum could you do a quick check against machine translator?

File: packages/component/src/Localization/nl-NL.js [hunk 1]
```diff
@@ -1,23 +1,71 @@
...
+  }
+}
+
 export default {
-  // FAILED_CONNECTION_NOTIFICATION: '',
-  // Do not localize {Retry}; it is a placeholder for "Retry". English translation should be, "Send failed. Retry."
-  SEND_FAILED_KEY: 'versturen mislukt, {Retry}.',
-  // SLOW_CONNECTION_NOTIFICATION: '',
+  FAILED_CONNECTION_NOTIFICATION: 'Verbinding maken niet mogelijk.',
+  SEND_FAILED_KEY: 'Versturen mislukt, {retry}.',
```
Comment (corinagum) [inline packages/component/src/Localization/nl-NL.js]: In English, the phrase "Send failed. Retry." is grammatically more correct than "Send failed, retry". In Dutch, is there a preference in this case?

Comment (corinagum) [reply packages/component/src/Localization/nl-NL.js]: if "Versturen mislukt. Opnieuw proberen" makes more sense, could you update this string to the following: 

`Versturen mislukt. {Retry}.'`

Comment (iMicknl) [reply packages/component/src/Localization/nl-NL.js]: `"Versturen mislukt, probeer opnieuw."` would be my preference. Will discuss this with another Dutch colleague tomorrow, but in Dutch we don't prefer cutting up those sentences.

Comment (corinagum) [reply packages/component/src/Localization/nl-NL.js]: Once I get confirmation from you that the single sentence is preferred, I will merge this PR in. Thanks for all your help!

Comment (iMicknl) [reply packages/component/src/Localization/nl-NL.js]: Confirmed. @corinagum

**Pull request merged**